examples/demo-prover/methods/build.rs

```diff
@@ -1,3 +1,14 @@
+use std::collections::HashMap;
+
 fn main() {
-    risc0_build::embed_methods();
+    #[cfg(not(feature = "bench"))]
```

Comment:
let's separate all the functionality hidden by #[cfg(feature = "bench")] in a separate method and feature guard this method.